src/doppler-types.ts

```diff
@@ -68,12 +74,14 @@ export interface PrepaidPack {
 }
 
 export interface FreePlan {
+  id: number;
```

Comment:
Does the freePlan have an id? how is it useful for use?

Reply:
To look for current plan by id. [Here](https://github.com/FromDoppler/doppler-webapp/pull/1074/files#diff-5710472294fa4b50ebeeb68baed8547aR103)

Reply:
But, you already have the plan, why did you search it by id?

```js
const monthlyPlan = planService.getPlanById(plan.idPlan, planList);
//                                          ^^^^ this is the plan
```

Reply:
I don´t have all plan data in session information, the name is wrong it should be session instead of plan perhaps. The only data I have for plan is: id, description, subscribers by month or emails, remaining credits, and that´s all, I need the fee for paths to work and to compare.

Reply:
I understand, it is a valid solution, but it is still making noise to me.

I would prefer something like `getPlanBySessionPlan()`, if we need that the session plan is the free plan, we should return the free plan, without worry about the ID. But, I am not sure, maybe this approach has worse problems.

Reply:
I will reformulate this, yes its weird, thanks!.